static/src/inline-svgs/commercial/survey-loudspeaker.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="65px" height="65px" viewBox="0 0 65 65" enable-background="new 0 0 65 65" xml:space="preserve"><g><circle fill="#BDBCBC" cx="32.5" cy="32.5" r="31.46"/><g><path fill="#00578A" d="M33.084,17.535l-1.509,0.692c0.326,9.358-11.986,16.589-11.986,16.589l3.232,7.09l1.592-0.617l1.522,8.416 c0,0,5.984,0.219,6.52-0.742c0,0,0.536-0.967,0.322-1.715c-0.213-0.752-0.964-0.537-1.178-1.287 c-0.213-0.75,0.536-0.963,0.322-1.713c-0.213-0.752-0.962-0.539-1.178-1.287c-0.213-0.752,0.538-0.965,0.324-1.717 c-0.071-0.246-0.295-0.717-0.562-1.236c4.847-0.932,8.121-2.146,11.375,0.656l1.5-0.686"/><path fill="#00578A" d="M16.864,36.828c-0.316,0.844,0.07,2.219,0.544,3.252s1.188,2.256,2.033,2.572 c0.842,0.314,2.002-0.916,2.002-0.916l-2.644-5.791C18.799,35.945,17.178,35.984,16.864,36.828z"/><path fill="#00578A" d="M48.244,37.748c-0.304,0.816-3.741,1.715-3.741,1.715L34.208,17.02c0,0,2.922-2.02,3.74-1.717 c0.409,0.152,3.387,5.41,5.896,10.881C46.353,31.652,48.395,37.338,48.244,37.748z"/></g></g></svg>
```

Comment:
It would be cool to reduce the decimal precision from three to one, ideally. If one garbles the graphic, then - two. SVGOMG is your friend :-)